ui/components/multichain/badge-status/badge-status.tsx

```diff
@@ -61,33 +52,20 @@ export const BadgeStatus: React.FC<BadgeStatusProps> = ({
           )
         }
       >
-        {
-          <AvatarAccount
-            borderColor={BorderColor.transparent}
-            size={AvatarAccountSize.Md}
-            address={address}
-            variant={
-              useBlockie
-                ? AvatarAccountVariant.Blockies
-                : AvatarAccountVariant.Jazzicon
-            }
-            marginInlineEnd={2}
-          />
-        }
+        {<PreferredAvatar address={address} className="flex" />}
       </BadgeWrapper>
     );
   }, [
     address,
     badgeBackgroundColor,
     badgeBorderColor,
     isConnectedAndNotActive,
-    useBlockie,
     showConnectedStatus,
   ]);
 
   return (
     <Box
-      className={classNames('multichain-badge-status', className)}
+      className={classNames('multichain-badge-status pr-1', className)}
```

Comment:
Do we need this padding when the consuming component already has gap? E.g. `AccountListItem`